src/master/jumpstart.md

```diff
@@ -22,19 +22,39 @@ In the most common scenario, you would perform write and read operations on the
 
 ## SDKs
 
+SDKs make it comfortable to talk to the server from your favourite language, without having to deal with details about how to talk to it.
+
+The most well-known and recommended immudb SDK is written in [Golang](https://golang.org/), but there are other SDKs available, both maintainer by the internal team and by the community.
+
+
+| Language         | Maintainer | Immdb version | link | Notes                                                                              |
+|-------------------|---------|------------------|-------------|-----------------------------------------------------------|
+| `go`               | immudb team  | 1.2.1       |     [link](https://github.com/codenotary/immudb)  |                                   |
```

Comment:
I'm wondering if it wouldn't be better to link to https://pkg.go.dev/github.com/codenotary/immudb/pkg/client - the whole immudb repository maybe too large and people may get confused.